src/workerd/jsg/modules.h

```diff
@@ -7,10 +7,12 @@
 #include <kj/filesystem.h>
 #include <kj/map.h>
 #include <workerd/jsg/modules.capnp.h>
+#include <workerd/util/thread-scopes.h>
 #include <workerd/jsg/observer.h>
 #include <set>
 #include "function.h"
 #include "promise.h"
+#include <workerd/server/autogate.h>
```

Comment:
Oy... hmm... the `autogate.h` stuff should probably actually be moved into `workerd/util`. We should not introduce any dependency on `workerd/server` from inside `workerd/jsg`.

Reply:
If you try to enable it, it also ends up using autogate too early:
```
error: Autogates not initialised, check for builtin-wasm will have no effect
```
I can open a separate PR to move this?

Reply:
Opened #1443.